orbit/cmd/orbit/orbit.go

```diff
@@ -575,6 +576,18 @@ func main() {
 			return fmt.Errorf("cleanup old files: %w", err)
 		}
 
+		// Kill any pre-existing instances of osqueryd (otherwise getHostInfo will fail
+		// because of osqueryd's lock over its database).
+		//
+		// This can happen for instance when orbit is killed via the Windows Task Manager
+		// (there's no SIGTERM on Windows) and the osqueryd child processes are left orphaned.
```

Comment:
Just curious, are the osquery extensions left orphaned as well? If so, does killing osqueryd parent kill the child extension processes?

Reply:
Good question, I'll test this today.

Reply:
Just tested an osquery-go extension and it terminates when the osquery process is not responding. (These extension SDKs usually perform pings at intervals.)